hack/test-end-to-end-docker.sh

```diff
@@ -10,14 +10,14 @@ set -o pipefail
 STARTTIME=$(date +%s)
 OS_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${OS_ROOT}/hack/util.sh"
+source "${OS_ROOT}/hack/lib/util/environment.sh"
 
 echo "[INFO] Starting containerized end-to-end test"
 
 unset KUBECONFIG
 
-TMPDIR="${TMPDIR:-"/tmp"}"
-BASETMPDIR="${BASETMPDIR:-${TMPDIR}/openshift-e2e-containerized}"
-setup_env_vars
+os::util::environment::setup_all_server_vars "test/end-to-end/docker/"
```

Comment:
the `test/` directory doesn't add much value either.  I'm fine with `test-end-to-end-docker` if you want `test` in there, but don't make me navigate deeper.